layouts/partials/sidebar.html

```diff
@@ -49,6 +49,16 @@ <h5>FEATURED TAGS</h5>
                        </span>
                    </a>
                </li>
+               {{ end }}
+			   {{ with .Site.Params.social.email }}
```

Comment:
Can we remove the leading space here?

Reply:
Sure. The space error seems caused by my evil text editor and did not appear locally. I will be rectifying it online, but please feel free to modify it if you still see it.